corgidrp/l2a_to_l2b.py

```diff
@@ -88,6 +88,35 @@ def convert_to_electrons(input_dataset):
 
     return None 
 
+def em_gain_division(input_dataset):
+    """
+    
+    Convert the data from electrons to photo-electrons by dividing the commanded em_gain. 
```

Comment:
This designation of unit change may be a bit confusing to the user.  Any residual noise (due to dark current or clock-induced charge, for example) would be electrons and not signal photo-electrons.  I would argue that the step that most closely approximates the specification of electrons to photo-electrons would be dark subtraction.  Dividing by gain would simply get rid of the scaling to the signal.

Reply:
For astronomers, we want to convert out of electrons at some point, so the point is to have a pipeline step that does this. We can put it after dark subtraction in the pipeline process if that makes sense. This should be done when every electron should be due to a photoelectron

Reply:
I agree.  For reference, Roman doesn't do that until after L4 (after field-distortion correction).

Reply:
Our requirements for CGI require L2b products be in photoelectrons. So we need to do this somewhere in the L2a->L2b step.

Reply:
Oh sorry, I was thinking "photon" instead of "photo-electron".  Yes, electron -> photo-electron would be via dark subtraction.